spec/adapters/alma_adapter_spec.rb

```diff
@@ -362,11 +362,23 @@
       expect(item[:in_temp_library]).to eq false
       expect(item[:temp_library_code]).to eq nil
 
-      # Make sure temp locations are handled
+      # Make sure temp locations are handled and the permanent location is preserved.
       availability = adapter.get_availability_holding(id: "9919392043506421", holding_id: "22105104420006421")
       item = availability.first
       expect(item[:in_temp_library]).to eq true
       expect(item[:temp_library_code]).to eq "online"
+      expect(item[:pickup_location_id]).to eq "pa"
+      expect(item[:pickup_location_code]).to eq "pa"
+
+      # Test an actual response. These values are not particularly meaningful, but to make sure we don't
+      # inadvertently change them when refactoring.
+      item_test = { barcode: "32101080920208", id: "23105104390006421", copy_number: "1", status: nil,
+                    on_reserve: nil, item_type: "Gen", pickup_location_id: "pa", pickup_location_code: "pa",
+                    location: "online$etasrcp", label: "ReCAP", in_temp_library: true, status_label: "Item in place",
```

Comment:
Can we change pickup_location_id: "pa" to -> 299? currently "PA" => "299".

Reply:
I don't think these ids will change https://github.com/pulibrary/voyager_helpers/blob/0ae062a25177187fb6b47ae19e29b45ea469acb0/lib/voyager_helpers/liberator.rb#L1325

Reply:
Can we make a separate issue for this? I don't think it's a refactor concern.

Reply:
a separate issue to change the value of the pickup_location_id in the test?

Reply:
Will changing the test not necessitate a code change?

Reply:
yes it will change. I will make a separate issue for that.

Reply:
opened https://github.com/pulibrary/bibdata/issues/1233

Reply:
Thanks!